app/(datasets)/data-catalog/catalog.tsx

```diff
@@ -0,0 +1,24 @@
+'use client';
+import React from 'react';
+import { CatalogView, useFiltersWithQS } from '../../lib';
```

Comment:
Optional for now, but ideal for long-term: We can optimize the paths here by adding an alias in the `tsconfig.json`:

```
{
  "compilerOptions": {
    "paths": {
      "@lib": ["app/lib"],
      "@lib/*": ["app/lib/*"]
    },
  }
}
```

so that we can replace relative imports with:

`import { CatalogView, useFiltersWithQS } from '@lib';`

Reply:
Is there any reason we are keeping the locally built lib here?